src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Services/SyntaxFacts/ISyntaxFacts.cs

```diff
@@ -495,6 +495,8 @@ internal interface ISyntaxFacts
 
         SyntaxToken? GetDeclarationIdentifierIfOverride(SyntaxToken token);
 
+        SyntaxToken? GetTargetIfControlFlow(SyntaxToken token);
```

Comment:
this isn't really a syntax-fact.  I would keep this in the feature itself.

Reply:
(the reason being hta tyou're effectively just encoding a feature here as no one really comes along and says "what token does this 'break' belong to?")